fluentd/run.sh

```diff
@@ -105,5 +105,8 @@ else
     echo > $CFG_DIR/dynamic/es-ops-copy-config.conf
 fi
 
-
-exec fluentd $fluentdargs
+if [[ $DEBUG ]] ; then
+    exec fluentd $fluentdargs > /var/log/fluentd.log 2>&1
```

Comment:
The problem is using filters like this for debugging, when you are trying to see the record "in flight":
```
<filter **>
  @type stdout
</filter>
```
This causes terrible feedback loops in fluentd/es.  The only solution is to _not_ output the log to stdout.